qds_sdk/commands.py

```diff
@@ -87,10 +87,26 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
         while not Command.is_done(cmd.status):
```

Comment:
I see. when command is finished api returns logs from cache / download from s3. In those cases err_length and tmp_length are 0 which are not correct indicators. Secondly is it safe to get all the logs once the command is finished, they could be very large. How are we handling this currently ? @msumit